apps/challenges/models.py

```diff
@@ -277,6 +277,8 @@ def __init__(self, *args, **kwargs):
     )
     # Id in the challenge config file. Needed to map the object to the value in the config file while updating through Github
     config_id = models.IntegerField(default=None, blank=True, null=True)
+    # Store the metadata for the default submission meta attributes of this challenge phase.
+    default_submission_meta_attributes = JSONField(default=None, blank=True, null=True)
```

Comment:
Can we please add the default value so that the current running challenges are not facing any issues as for current challenges, this field will be None and no fields will get displayed.

Reply:
@RishabhJain2018 default value is null which is already handled on UI side.